14/umbraco-cms/reference/security/external-login-providers.md

```diff
@@ -177,9 +177,9 @@ using Umbraco.Cms.Web.BackOffice.Security;
 
 namespace MyUmbracoProject.CustomAuthentication;
 
-public class ProviderBackOfficeExternalLoginProviderOptions : IConfigureNamedOptions<BackOfficeExternalLoginProviderOptions>
+public class GenericBackOfficeExternalLoginProviderOptions : IConfigureNamedOptions<BackOfficeExternalLoginProviderOptions>
```

Comment:
In this code example am getting 2 errors:
- regarding Icon:

![image](https://github.com/umbraco/UmbracoDocs/assets/83591955/81355d01-e8ea-4a2f-be2a-b98942ac3a2a)

- regarding AutoRedirectLoginToExternalProvider:

![image](https://github.com/umbraco/UmbracoDocs/assets/83591955/039c223e-ae6a-4b31-aa80-f878609f1f00)

Reply:
The Generic one will not work, its just a code example it has never worked as there is no actual generic provider (I tried the old example just to be sure 😅

Reply:
these 2 do not exist anymore and can be removed